session.go

```diff
@@ -1484,6 +1488,11 @@ func (b *Batch) WithContext(ctx context.Context) *Batch {
 	return b
 }
 
+// GetContext satisfies the ExecutableQuery interface.
+func (b *Batch) GetContext() context.Context {
```

Comment:
Call this `Context()` along with the one on Query to me more inline with go style.

Reply:
all good now